src/blog/2024/08/comparing-dashboard-2-with-uibuilder.md

```diff
@@ -0,0 +1,233 @@
+
+## UI-Builder
+
+UI-Builder also offers a number of widgets, but these are not as user-friendly as those in Dashboard 2.0. Users must send a JSON config object, which can be complex for new users who lack knowledge of HTML/CSS. Additionally, handling widget data requires using UI-Builder's methods, which can further increase complexity. However, UI-Builder's strength lies in its flexibility, allowing any HTML element to be used as a component, and it also supports the W3C standard web components. Despite this, users need to perform a lot of additional tasks to get everything set up and will have the hassle of writing things.
+
+## How much dynamic insertion or update it support?
+
+### Node-RED Dashboard
+
+Node-RED Dashboard 2.0 supports dynamically updating the UI via the msg object. Each UI widget supports the `msg.ui_update` property with a JSON object that allows you to update the UI components dynamically. For example, you can update form fields based on user input, dynamically insert or update options in a dropdown, or change the appearance of the UI by sending css classes.
+
+### Ui-builder
+
+UI-Builder also supports dynamic UI updates. Similar to Node-RED Dashboard, you can use messages to control the state and content of UI elements.
+
+## Is creating visualizations easy in the Dashboard? ?
+
+### Node-RED Dashboard 2.0
+
+Visualization is a core use case for Node-RED, and Node-RED Dashboard 2.0 makes it easy with its built-in [chart](https://dashboard.flowfuse.com/nodes/widgets/ui-chart.html) and [gauge widgets](https://dashboard.flowfuse.com/nodes/widgets/ui-gauge.html). The chart widgets support various types, including line, bar, scatter, pie, and donut charts, while the gauge widgets offer options like tile, 3/4 gauge, and half gauge. This range of built-in options simplifies the creation of visualizations. Additionally, you can use Vuetify or other third-party libraries in the ui-template node if you need different types of charts.
+
+### UI-Builder
+
+UI-Builder does not provide built-in charting options. However, you can add charts using third-party libraries, which can make the visualization setup more complex and time-consuming. This approach requires additional effort to integrate and configure the libraries, potentially increasing the complexity of creating visualizations.
+
+## How Fast and Easy is It to Communicate with Node-RED Instances?
+
+### Node-RED Dashboard 2.0
+
+Node-RED Dashboard 2.0 uses WebSockets for communication with Node-RED, enabling real-time and faster interactions. Most widgets uses the standard `msg.payload` for communication that makes it super easy to send the data from Node-RED to client and vice-versa.
+
+### Ui-builder 
+
+The ui-builder also employs WebSockets for communication, providing real-time and fast interactions. However, it utilizes different msg properties, which can be complex for users and might increase the number of nodes unnecessarily for changing the property names.
```

Comment:
I'd remove this section, not sure it's clear to read or adds a valuable comparison